modules/tinyexr/image_saver_tinyexr.h

```diff
@@ -0,0 +1,8 @@
+#ifndef IMAGE_SAVER_TINYEXR_H
```

Comment:
You should add the Godot license header here and in the cpp.
Note that line 2 should include the filename, and that is has a trailing `*/` that should be kept properly aligned.